python/cugraph/cugraph/dask/common/input_utils.py

```diff
@@ -103,6 +103,13 @@ def create(cls, data, client=None, batch_enabled=False):
         else:
             raise Exception("Graph data must be dask-cudf dataframe")
 
+        # FIXME: This is a workaround for the fact that the
```

Comment:
Could you add a GitHub issue to track this?

Reply:
Removed this as no longer using